src/app/rpm-ostree-countme.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Weekly rpm-ostree Count Me reporting
+Documentation=man:rpm-ostree-countme(1)
+ConditionPathExists=/run/ostree-booted
+
+[Service]
+Type=simple
+User=rpm-ostree
+DynamicUser=yes
```

Comment:
That said, this will break in the specific (but notable, since RHEL works this way) scenario where access to the target repository is gated by a client side certificate and we want those files owned by root and mode `0600`.  This is one instance of a larger problem with trying to drop privileges inside parts of rpm-ostree.

Reply:
Indeed, I was mostly thinking about FCOS support first.

Reply:
Let's start with an unprivileged user as we can add a system snippet to move it to root if needed in some special cases?